README.md

```diff
@@ -1,16 +1,21 @@
-# Title
+# Introduction to Getting Started with SQL
 
-## Learning Goals
+Just as humans have brains, and computers have hard drives that store
+information, most programs have a storage system that needs to be communicated
+with. How do we store and find stored data? In databases, and using SQL. It can
+be used with a ton of different databases like MySQL, Postgres, etc. It powers
+the logic behind server-side frameworks. SQL has many variations, like SQLite,
```

Comment:
maybe 'SQL has many variations, but for our lessons, we will be using SQLite (link). While there are some minor differences we'll note later on, the fundamentals stay the same.'